spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md

```diff
@@ -205,3 +213,53 @@ This approach is depicted in the following Figure that shows an overview of the
   The unbonding operation is completed later once the provider CCV module registered maturity notifications for `VSC2` from all consumer chains.
 
 ![Completion of Unbonding Operations](./figures/ccv-unbonding-overview.png?raw=true)
+
+### Consumer Initiated Slashing
+[&uparrow; Back to Outline](#outline)
+
+For the [Security Model](#security-model) to be preserved, misbehaving validators MUST be slashed (and MAY be jailed, i.e., removed from the validator set). 
+A prerequisite to slash validators is to receive valid evidence of their misbehavior. 
+Thus, when slashing a validator, we distinguish between three events and the heights when they occur:
+- `infractionHeight`, the height at which the misbehavior (or infraction) happened;
+- `evidenceHeight`, the height at which the evidence of misbehavior is received;
+- `slashingHeight`, the height at which the validator is slashed (and jailed). 
+
+> **Note**: In the context of single-chain validation, usually `evidenceHeight = slashingHeight`. 
+
+The [Security Model](#security-model) guarantees that any misbehaving validator can be slashed for at least the unbonding period, 
+i.e., as long as that validator's tokens are not unbonded yet, they can be slashed. 
+However, if the tokens start unbonding before `infractionHeight`, i.e., the tokens did not contribute to the voting power that committed the infraction, 
+then the tokens MUST NOT be slashed.
+
+In the context of CCV, validators (with tokens bonded on the provider chain) MUST be slashed for infractions committed on the consumer chains. 
+Thus, although the infractions are committed on the consumer chains and evidence of these infractions is submitted to the consumer chains, the slashing happens on the provider chain. As a result, the Consumer Initiated Slashing operation requires, for every consumer chain, a mapping from consumer chain block heights to provider chain block heights.
```

Comment:
Could use a diagram of the mapping as a supplement to the current diagram that illustrates more of the mechanic. Something really simple like 
```
        Consumer                  IBC               Provider

cHeight <--- VSCtoCHeight[] ---> VSCid <--- VSCtoPHeight[] ---> pHeight
```

Reply:
I'm not sure I understand the diagram. From the names, both mappings seems to map from VSCs to heights. From the arrows, I'm not sure what's the input and what's the output. Also, why is vscId under IBC?  

Do you mean sth like this?
```
        Consumer                                Provider

cHeight ---> HtoVSC[] ---> VSCid ---> VSCtoH[] ---> pHeight
```
I don't really see how this is more helpful than the "Mapping consumer heights to provider heights" figure and its explanation.

Reply:
@jtremback I added an extra diagram that provides the intuition  behind the mapping.